packages/collector/src/logger.js

```diff
@@ -37,6 +46,7 @@ exports.init = function init(config, isReInit) {
     // we create later on.
     parentLogger = bunyan.createLogger({
       name: '@instana/collector',
+      thread: threadId,
```

Comment:
Adding the thread ID to the log object is not strictly necessary but we have the PID in there, too, and I think it might be pretty helpful when trouble shooting scenarios involving worker threads in the future.